smart-contracts/contracts/CosmosBridge.sol

```diff
@@ -292,21 +378,42 @@ contract CosmosBridge is CosmosBridgeStorage, Oracle {
         );
     }
 
-    function _isManagedToken(address tokenAddress) private returns(bool) {
+    /**
+     * @dev Verifies if `tokenAddress` is a known token
+     * @param tokenAddress The address of the token
+     * @return Boolean: is `tokenAddress` a known token?
+     */
+    function _isManagedToken(address tokenAddress) private view returns(bool) {
```

Comment:
Let's change this function name to `_isDoublePeggedToken` as this is really checking to see if an asset is double pegged or not.